creator-node/src/redis.js

```diff
@@ -1,36 +1,123 @@
+/**
```

Comment:
diff is confusing - better way to view this might be to see what's currently on master for RedisLock: https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/redis.js

Changes:
- replaces RedisLock class with WalletWriteLock object